server/api.go

```diff
@@ -400,6 +400,18 @@ func (a *API) oauthRedirectHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if err := a.p.store.StoreUserInWhitelist(mmUserID); err != nil {
+		if !strings.Contains(err.Error(), "Duplicate entry") {
+			a.p.API.LogError("Unable to store the user in whitelist", "UserID", mmUserID, "Error", err.Error())
+			if err = a.p.store.SetUserInfo(mmUserID, msteamsUser.ID, nil); err != nil {
+				a.p.API.LogError("Unable to delete the OAuth token for user", "UserID", mmUserID, "Error", err.Error())
```

Comment:
should we return from here as the user is connected, still we are returning the error of not able to complete oauth?